pennylane/_qubit_device.py

```diff
@@ -656,9 +656,10 @@ def marginal_prob(self, prob, wires=None):
         # it corresponds to the orders of the wires passed.
         num_wires = len(device_wires)
         basis_states = self.generate_basis_states(num_wires)
-        perm = np.ravel_multi_index(
-            basis_states[:, np.argsort(np.argsort(device_wires))].T, [2] * len(device_wires)
-        )
+        basis_states = basis_states[:, np.argsort(np.argsort(device_wires))].T
```

Comment:
Is the double argsort here redundant?

Reply:
I don't believe so, but I'll need to double check tomorrow.

Reply:
I checked and it's correct as is.